### DISCUSSION TITLE
Most releases are "Version bump only for package @react-navigation/xxx"

### DISCUSSION THREAD

**Discussion by hatem-72** [react-navigation/react-navigation#11093] [General]
Hello 👋
First of all I would like to thank the maintainers for their continuous work on this library.

I wanted to talk about the release process that I find a bit annoying for some time since most of the releases are `Version bump only for package @react-navigation/xxx`.
I guess this is due to the monorepo approach of the project but I would have liked to know if it was absolutely necessary to release all the packages each time a minor modification was made, even if it is for one single minor package only.
I like to keep my dependencies up-to-date, but it takes time and effort. With this approach it's getting hard to read the release notes since most of the updates do not provide any really useful content.

**Comment by satya164** [comment]
Unfortunately, that's how Lerna works and we haven't found a solution. Feel free to suggest ideas to handle it better.

**Reply by hatem-72** [reply]
Yes I know that's not how Lerna works 😐.
One solution would be to manually publish each package using `npm publish`. But I guess it will bring more problems than it solves.

**Reply by ghiscoding** [reply]
hi there, Lerna-Lite maintainer here... @satya164, @hatem-72 I just found and read your discussion today and I believe that I could maybe provide some help with this problem. I created this Lerna-Lite PR [feat(version): add `--skip-bump-only-release` to avoid empty gh releases](https://github.com/lerna-lite/lerna-lite/pull/555) which should help with your use case. 

Note however that this new option will only be available in the Lerna-Lite v2.0 and higher, I just released a v2.0.0-alpha release just a couple days ago (see ~[v2.0.0-alpha](https://github.com/lerna-lite/lerna-lite/releases/tag/v2.0.0-alpha.2)~ official [v2.0.0](https://github.com/lerna-lite/lerna-lite/releases/tag/v2.0.0) release) and I'm not planning to work on v1.x anymore, the final v2.0 release ~should ship in a week or so~ is now available and it will include this PR and new feature. Also note that I do not use the `independent` mode myself and I only added unit tests for this new option but I wasn't able to test it with a real repo, however I'm pretty certain that the tests are correct and covers the use case, It could be nice to get a PR review though, I won't merge the PR until a few more days so feedback would be welcome. Let me know if that could help with your use case.

**Comment by raajnadar** [comment]
You can safely upgrade to the latest minor version as there will be no breaking changes, which should not block you from upgrading & in most cases you don’t need to read the change logs for minor version bump

**Reply by hatem-72** [reply]
Yes of course, that's what I've been doing for months. It's not a huge concern but I'm pointing out the bad DX here.

Not reading the release notes seems rather unwise advice to me. I mean updating dependencies in a JS project has never been easy and without friction or bad surprises. Even for minor upgrades.
